mdx/guides/build-your-first-cfl-enabled-smart-contract.mdx

```diff
@@ -0,0 +1,270 @@
+To learn more about contract fillable liquidity at a high level, read this guide [here](/docs/guides/abcs-of-contract-fillable-liquidity). If you are comfortable with the concepts of contract fillable liquidity (CFL), feel free to continue this guide and create your first CFL integration with 0x.
+
+## Overview
+
+After you complete this guide, you will have a smart-contract that swaps Ether for DAI, powered by 0x API.
+
+At a high level, in this guide we will complete 3 main steps:
+
+1. Leverage 0x API `/swap/v0/quote` endpoint to fetch a quote for a specified swap.
+2. Send calldata for a 0x protocol contract call provided in the response of `/swap/v0/quote` to your smart contract.
+3. Fill 0x orders in your smart contract.
+
+Let’s get rollin’!
+
+---
+
+## Setting up your off-chain web3 script 👩🏻‍🔬
+
+We will be using our **getting started with 0x API** starter project found [here](https://github.com/0xProject/0x-api-starter-guide-code). The repo has a basic setup for smart contract development and interacting with them on ganache.
```

Comment:
I would not use the "found here" pattern, and just try to work the links in. So here just make "getting started with 0x API" the link.